x-pack/plugin/inference/qa/inference-service-tests/src/javaRestTest/java/org/elasticsearch/xpack/inference/InferenceBaseRestTest.java

```diff
@@ -64,6 +64,26 @@ static String mockServiceModelConfig() {
             """;
     }
 
+    protected Map<String, Object> downloadElserBlocking() throws IOException {
+        String endpoint = "_ml/trained_models/.elser_model_2?wait_for_completion=true";
+        if (System.getProperty("os.arch").contains("amd64")) {
```

Comment:
If you look in `Platforms.java` you'll see there's code that treats both `amd64` and `x86_64` the same: https://github.com/elastic/elasticsearch/blob/12ad399c488f0cc60e19b5e1b29c6d569cb4351a/server/src/main/java/org/elasticsearch/plugins/Platforms.java#L64-L65

Maybe Java is more consistent about these constants now, but I think it's better to reuse the code from `Platforms.java` rather than duplicate here.

```suggestion
        if ("linux-x86_64".equals(Platforms.PLATFORM_NAME)) {
```

Reply:
Thanks Dave, for some reason I didn't think I would be able to use the code in the Platforms plugin without adding a dependency, but it did just work.